### PR TITLE
some more keys (end with alienDeployments)

### PR DESCRIPTION
**Pull request by Finnik723**

### CODE REVIEW

File: src/rulesetDefinitionChecker.ts [hunk 1]
```diff
@@ -97,10 +97,6 @@ export class RulesetDefinitionChecker {
     };
 
     private typeLinks: {[key: string]: string[]} = {
-        'armors.corpseBattle': ['items'],
-        'alienDeployments.data[].itemSets[]': ['items'], // could check if it's a weapon?
```
Comment (pedroterzero) [inline src/rulesetDefinitionChecker.ts:101]: @Finnik723 do I add this one back in? Or was it incorrect?

File: src/rulesetDefinitionChecker.ts [hunk 2]
```diff
@@ -97,10 +97,6 @@ export class RulesetDefinitionChecker {
...
-        'items.categories': ['itemCategories'],
-        'items.tags': ['extended.tags.RuleItem'],
```
Comment (pedroterzero) [inline src/rulesetDefinitionChecker.ts:103]: @Finnik723 do I add this one back in (103)? Or was it incorrect?

**Comment by pedroterzero** [comment]
Looks good apart from my two questions, but those can be done later.

Merging, thanks once again!

File: src/rulesetDefinitionChecker.ts (hunk 1 above)
Comment (Finnik723) [reply src/rulesetDefinitionChecker.ts:101]: It can be any item, im using that to give extra mission loot

**Pull request merged**